src/eager/ort_eager.cpp

```diff
@@ -65,6 +66,9 @@ PYBIND11_MODULE(torch_ort, torch_ort_module) {
   torch_ort_module.def("ort_from_dlpack", [](py::object dlpack_tensor) {
     return ORTTensor_FromDLPack(dlpack_tensor);
   });
+
+  auto customop_module = torch_ort_module.def_submodule("custom_ops");
```

Comment:
we can use some existing custom op in onnxruntime cpu provider as a test case for this api.